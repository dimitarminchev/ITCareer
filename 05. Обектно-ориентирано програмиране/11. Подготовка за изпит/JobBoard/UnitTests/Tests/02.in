AddCategory It
AddJobOffer It BackEndDev CareerSoft 6000 remote True
End