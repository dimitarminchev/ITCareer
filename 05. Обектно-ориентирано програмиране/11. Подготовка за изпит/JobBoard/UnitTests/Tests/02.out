Created Category It!
Created JobOffer BackEndDev in Category It!